--- qtiplot-0.9.8.5/qtiplot/qtiplot.pro.compile	2011-02-14 12:21:47.000000000 +0100
+++ qtiplot-0.9.8.5/qtiplot/qtiplot.pro	2011-03-19 05:57:41.000000000 +0100
@@ -42,7 +42,6 @@
 	INSTALLS        += target
 	INSTALLS        += translations
 	INSTALLS        += manual
-	INSTALLS        += documentation
 	unix:INSTALLS        += man
 
 	unix: INSTALLBASE = /usr
@@ -106,9 +105,9 @@
 					translations/qtiplot_sv.qm
 
 isEmpty(LUPDATE): LUPDATE = lupdate
-#system($$LUPDATE -verbose qtiplot.pro)
+system($$LUPDATE -verbose qtiplot.pro)
 isEmpty(LRELEASE): LRELEASE = lrelease
-#system($$LRELEASE -verbose qtiplot.pro)
+system($$LRELEASE -verbose qtiplot.pro)
 
 ###################### DOCUMENTATION ########################################
 
@@ -151,7 +150,7 @@
   				    qtiUtil.py \
   				    qti_wordlist.txt \
 
-  	unix: pythonconfig.path = /usr/local/qtiplot
+  	unix: pythonconfig.path = /usr/share/qtiplot
   	win32: pythonconfig.path = $$INSTALLBASE
   	DEFINES += PYTHON_CONFIG_PATH="\\\"$$replace(pythonconfig.path," ","\ ")\\\"
   }
@@ -230,6 +229,7 @@
 }
 
 QT          += sql
+win32 {
 LIBS        += src/plugins/libQtiPlotdBasePlugin.a
 LIBS        += src/plugins/libQtiPlotDatabasePlugin.a
 LIBS        += src/plugins/libQtiPlotCsvPlugin.a
@@ -241,3 +241,4 @@
 LIBS        += ../3rdparty/quazip/lib/libquazip.a
 LIBS        += ../3rdparty/EmfEngine/libEmfEngine.a
 LIBS        += -lgdiplus
+}
--- qtiplot-0.9.8.5/build.conf.compile	2011-03-19 05:39:06.000000000 +0100
+++ qtiplot-0.9.8.5/build.conf	2011-03-19 05:39:06.000000000 +0100
@@ -0,0 +1,148 @@
+isEmpty( QTI_ROOT ) {
+  message( "each file including this config needs to set QTI_ROOT to the dir containing this file!" )
+}
+
+##########################################################
+##     System specific configuration
+##########################################################
+
+# Global include path which is always added at the end of the INCLUDEPATH
+SYS_INCLUDEPATH = /usr/include
+# Global lib path and libs which is ls always added at the end of LIBS
+SYS_LIBS = -L/usr/lib64 -L/usr/lib
+
+##########################################################
+## zlib (http://www.zlib.net/)
+##########################################################
+
+# include path. leave it blank to use SYS_INCLUDE
+ZLIB_INCLUDEPATH =
+
+##########################################################
+## muParser (http://muparser.sourceforge.net/)
+##########################################################
+
+# include path. leave it blank to use SYS_INCLUDE
+MUPARSER_INCLUDEPATH = 
+# link statically against a copy in 3rdparty/
+#MUPARSER_LIBS = $$QTI_ROOT/3rdparty/muparser/lib/libmuparser.a
+# or dynamically against a system-wide installation
+MUPARSER_LIBS = -lmuparser
+
+##########################################################
+## GNU Sientific Library (http://www.gnu.org/software/gsl/)
+##########################################################
+
+# include path. leave it blank to use SYS_INCLUDE
+GSL_INCLUDEPATH = 
+# link statically against a copy in 3rdparty/
+#GSL_LIBS = $$QTI_ROOT/3rdparty/gsl/lib/libgsl.a \
+#           $$QTI_ROOT/3rdparty/gsl/lib/libgslcblas.a
+# or dynamically against a system-wide installation
+GSL_LIBS = -lgsl -lgslcblas
+
+##########################################################
+## QWT - use local copy till upstream catches up
+# http://qwt.sourceforge.net/index.html
+##########################################################
+
+# include path.
+QWT_INCLUDEPATH = $$QTI_ROOT/3rdparty/qwt/src
+# link locally against a copy in 3rdparty/
+QWT_LIBS = $$QTI_ROOT/3rdparty/qwt/lib/libqwt.a
+
+##########################################################
+## QwtPlot3D - use local copy till upstream catches up
+# http://qwtplot3d.sourceforge.net/
+##########################################################
+
+# include path.
+QWT3D_INCLUDEPATH = $$QTI_ROOT/3rdparty/qwtplot3d/include
+# link locally against a copy in 3rdparty/
+win32:QWT3D_LIBS = $$QTI_ROOT/3rdparty/qwtplot3d/lib/qwtplot3d.dll
+unix:QWT3D_LIBS = $$QTI_ROOT/3rdparty/qwtplot3d/lib/libqwtplot3d.a
+
+##########################################################
+## libpng - optional. you don't have to set these variables
+##########################################################
+
+# include path. leave it blank to use SYS_INCLUDE
+LIBPNG_INCLUDEPATH = 
+# link statically against a copy in 3rdparty/
+#LIBPNG_LIBS = $$QTI_ROOT/3rdparty/libpng/libpng.a
+# or dynamically against a system-wide installation
+LIBPNG_LIBS = -lpng
+
+##########################################################
+## QTeXEngine - optional. you don't have to set these variables
+# http://soft.proindependent.com/qtexengine/
+##########################################################
+
+# include path.
+#TEX_ENGINE_INCLUDEPATH = $$QTI_ROOT/3rdparty/QTeXEngine/src
+# link locally against a copy in 3rdparty/
+#TEX_ENGINE_LIBS = $$QTI_ROOT/3rdparty/QTeXEngine/libQTeXEngine.a
+
+##########################################################
+## ALGLIB (2.6) - optional. you don't have to set these variables
+# http://www.alglib.net/
+##########################################################
+
+# include path.
+#ALGLIB_INCLUDEPATH = $$QTI_ROOT/3rdparty/alglib/
+# link locally against a copy in 3rdparty/
+#ALGLIB_LIBS = $$QTI_ROOT/3rdparty/alglib/libalglib.a
+
+##########################################################
+## TAMUANOVA - optional. you don't have to set these variables
+# http://www.stat.tamu.edu/~aredd/tamuanova/
+##########################################################
+
+# include path.
+TAMUANOVA_INCLUDEPATH = $$QTI_ROOT/3rdparty/tamu_anova/
+# link locally against a copy in 3rdparty/
+TAMUANOVA_LIBS = $$QTI_ROOT/3rdparty/tamu_anova/libtamuanova.a
+
+##########################################################
+## python - only used if python is needed
+##########################################################
+
+# the python interpreter to use
+# (unix only, windows will use what ever is configured to execute .py files!)
+PYTHON = python
+
+##########################################################
+## Qt tools - allows to use specific versions
+##########################################################
+
+LUPDATE = lupdate
+LRELEASE = lrelease
+
+############################################################
+##  Target specific configuration: configure Qtiplot itself
+############################################################
+
+contains( TARGET, qtiplot ) {
+  # building without muParser doesn't work yet
+  SCRIPTING_LANGS += muParser
+  SCRIPTING_LANGS += Python
+
+  # a console displaying output of scripts; particularly useful on Windows
+  # where running QtiPlot from a terminal is inconvenient
+  DEFINES         += SCRIPTING_CONSOLE
+
+  #DEFINES         += QTIPLOT_DEMO
+
+  # Uncomment the following line if you want to perform a custom installation using the *.path variables defined in ./qtiplot.pro.
+  CONFIG          += CustomInstall
+
+  # Uncomment the following line if you want to build QtiPlot as a browser plugin (not working on Internet Explorer).
+  #CONFIG          += BrowserPlugin
+  
+  CONFIG          += release
+  #CONFIG          += debug
+  
+  # Uncomment the following line if you want to link statically against Qt.
+  #CONFIG           += StaticBuild
+  #win32: CONFIG   += console
+}
--- qtiplot-0.9.8.5/3rdparty/qwtplot3d/qwtplot3d.pro.compile	2011-01-25 06:07:31.000000000 +0100
+++ qtiplot-0.9.8.5/3rdparty/qwtplot3d/qwtplot3d.pro	2011-03-19 05:39:06.000000000 +0100
@@ -10,7 +10,7 @@
 DEPENDPATH       = include src
 QT              += opengl
 
-CONFIG          += dll
+CONFIG          += staticlib
 win32:CONFIG    += exceptions
 win32:dll:DEFINES    += QT_DLL QWT3D_DLL QWT3D_MAKEDLL
 win32:QMAKE_CXXFLAGS += $$QMAKE_CFLAGS_STL
@@ -38,5 +38,5 @@
 }
 
 # install
-target.path = lib
-INSTALLS += target
+#target.path = lib
+#INSTALLS += target
--- qtiplot-0.9.8.5/qtiplot.pro.compile	2010-09-20 19:08:10.000000000 +0200
+++ qtiplot-0.9.8.5/qtiplot.pro	2011-03-19 05:39:06.000000000 +0100
@@ -1,7 +1,6 @@
 TEMPLATE = subdirs
 
 SUBDIRS = fitPlugins \
-	    manual \
 	    3rdparty/qwt \
 		3rdparty/qwtplot3d \
         qtiplot
